state/src/impls/top_level.rs

```diff
@@ -143,6 +144,11 @@ impl TopStateView for TopLevelState {
         let trie = TrieFactory::readonly(db.as_hashdb(), &self.root)?;
         Ok(self.top_cache.ibc_data(key, &trie)?.map(Into::into))
     }
+    fn ibc_data_proof(&self, key: &H256) -> TrieResult<(CryptoProofUnit, CryptoProof)> {
```

Comment:
We are adding a new line between function definition.

Reply:
Ok. I didn't run the `fmt` this time.